### PR TITLE
Use real files for USWDS tests, allowing them to pass CSP checks.

### PR DESCRIPTION
**Pull request by devin-civi**
### Description

Use real pngs to work around the CSP bug. The main motivating reason behind this change is so that we can test Multiple file uploads once they switch to use USWDS.

### CODE REVIEW

**Comment by devin-civi** [comment]
* **#8382** <a href="https://app.graphite.dev/github/pr/civiform/civiform/8382?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @devin-civi and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: browser-test/src/assets/civiform-logo-alternate.png
Comment (gwendolyngoetz) [inline browser-test/src/assets/civiform-logo-alternate.png:1]: (blocking) Make the images visually different from each other so it's easier to tell if that something has changed or if the same image was uploaded.

Please so also rename the files to something related to their purpose. I can see down the road questions on "why do we have a logo image here".

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: browser-test/src/assets/civiform-logo-alternate.png
Comment (devin-civi) [reply browser-test/src/assets/civiform-logo-alternate.png:1]: Done

File: browser-test/src/assets/file-upload-second.png
Comment (gwendolyngoetz) [inline browser-test/src/assets/file-upload-second.png:1]: Love it!

**Review by gwendolyngoetz — APPROVED** [review]

**Review by jeffpw-goog — APPROVED** [review]
Left you a comment in slack but otherwise lgtm. Thanks for making this better!

**Pull request merged**